tests/unit/test_slack.py

```
@@ -6,12 +6,29 @@
 
 
 class TestFormatCompleteMessage(unittest.TestCase):
+    def test_suffix_set_correctly_for_one_run(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_suffix_set_correctly_for_one_run`

Add return type annotation: `None`

(ANN201)